packages/server/jest.config.ts

```diff
@@ -60,6 +54,12 @@ const config: Config.InitialOptions = {
     "!src/db/views/staticViews.*",
     "!src/**/*.spec.{js,ts}",
     "!src/tests/**/*.{js,ts}",
+    // The use of coverage in the JS runner bundles breaks tests
+    "!src/jsRunner/bundles/**/*.{js,ts}",
```

Comment:
I do think we should just set the whole `jsRunner` directory to be excluded, within this directory there is a lot of danger when it comes to the code coverage, and excluding it I feel is pretty understandable.